tests/python/unittest/test_tir_analysis_estimate_tir_flops.py

```diff
@@ -48,5 +50,16 @@ def test_te_workload(workload, flops):
     assert float(flops) == estimate_tir_flops(mod)
 
 
+@T.prim_func
+def flops_with_let(a: T.Buffer[16, "float32"]):
+    for i in range(8):
+        j = i + 8
+        a[j] = a[i]
+
+
+def test_flops_with_let():
+    estimate_tir_flops(IRModule({"main": flops_with_let}))
```

Comment:
Could you please add an `assert` to check the flops number?

Reply:
Good point, done.